### ISSUE TITLE
New admin layout - steps before merge

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#571]
The [admin-layout-branch patch](https://github.com/textpattern/textpattern/tree/admin-layout-update) that has been around for a year or so is nearing at stage a where it can be merged into master. This issue exists to track what still needs to be done before the merge (list will be updated and added to as necessary):

- [x] flexbox is causing bugs in Firefox and Chrome that need to be fixed.
- [x] rearrange the users list page to new layout as used on all other list panels (i.e. the alert messages specifically, but also code structure as it is inconsistent at the moment.
- [x] fix up the vertical jQuery UI list style, as used on new preferences panel layout.
- [x] fix the various form field input and value pairing layouts and ensure constant code structure throughout UI.
- [x] expanding `textrarea` fields instead of explicitly defined ones we current have (TBC)?
- [x] setup panels need some fix-ups to layout.
- [x] better layout for image edit panel - [see proposal here](http://www.philwareham.co.uk/textpattern/theme/hive/mockups/secondary/content-image_edit.html).
- [x] better layout for file edit panel (TBC).
- [x] remove redundant txp-container `div`.
- [x] move messagepane out of header area.
- [x] preferences page heading needs to move out of 1st column, to sit above.
- [x] investigate sticky footer issues in IE10&11/Safari 7&8 - [see here](http://forum.textpattern.com/viewtopic.php?pid=296600#p296600) for info.
- [x] Refactor switcher lists and jQuery Tabs so that `a` links are block elements (UX problem).
- [x] get some willing testers from the forum to give it a spin.

Other layout work will follow in smaller patches (hopefully including a grid view for images).

**Comment by philwareham** [comment]
Also, on merge we can close issue #553, #580, #578, #352, #325

**Comment by phiw13** [comment]
1. Is moving `#messagepane` out of `.txp-header` still on the radar (issue #237)?
2. You probably know that, but on the Write panel there is a huge layout difference for the data & time pane, depending on whether you look at a blank panel (or unsaved article) or at an already saved article.
3. Question: on some panels, the content is still wrapped inside a `div.txp-container` (edit panels, preferences panel). I don’t think that makes sense anymore. That `div` was originally introduced in the days of nested tables both as a way to control layout and an eventual hook for extension authors.

**Comment by philwareham** [comment]
Hi Philippe. Thanks, I'll answer those:

1. Yes moving the messagepane is still planned, I just haven't done it yet. It probably should come within this branch so I will add to the checklist above.
2. I've not completed the move of the label/input elements to new HTML structure yet, that is no.3 on checklist above. Date and time fields should look like rest of layout after that.
3. Yep that div was from nested grid - let's look at removing it then (wasn't sure if it had value to theme authors or not). Will add to list too.

**Comment by philwareham** [comment]
@Bloke do you want to take charge of point 1 above, so I can concentrate on the other issues. It's basically making the admin (users) page use the same code roughly as the other panels.

Then we are pretty close to merging this baby into the master and waiting for the plaudits/curses/tears/anger from our alpha testers :)

**Comment by Bloke** [comment]
Sure thing. I'll do it this weekend.

**Comment by Bloke** [comment]
Admin->Users panel [is now consistent](https://github.com/textpattern/textpattern/commit/a1584d7fd020aa0f6398b26434da577749b2310b), I think. Tick that one off if you're happy or tweak to taste.

**Comment by philwareham** [comment]
Thanks Stef, ticked that one off now.

**Comment by philwareham** [comment]
The pull request is activated, so this issue can close now.
